src/utils/utils.ts

```diff
@@ -0,0 +1,10 @@
+import { K8sModel } from '@openshift-console/dynamic-plugin-sdk/lib/api/common-types';
+
+const modelToRef = (obj: K8sModel) => `${obj.apiGroup}~${obj.apiVersion}~${obj.kind}`;
```

Comment:
Where do we need to use references? We've been trying to avoid exposing that in the SDK, so we should look at what gap we have.

Reply:
It is used for `ListPageCreate` component from `@dynamic-plugin-sdk` which requires `GroupVersionKind` string (in our case `stable.example.com~v1~CronTab`)

Reply:
@raspbeep Please open a bug for this component. We shouldn't be exposing `GroupVersionKind` strings in the API, so we'll want to deprecate this property and replace it with a prop where these values are specified separately.